src/ln/channelmanager.rs

```diff
@@ -173,6 +173,48 @@ impl MsgHandleErrInternal {
 	fn from_no_close(err: msgs::HandleError) -> Self {
 		Self { err, needs_channel_force_close: false }
 	}
+	#[inline]
+	fn from_chan_no_close(err: ChannelError, channel_id: [u8; 32]) -> Self {
+		Self {
+			err: match err {
+				ChannelError::Ignore(msg) => HandleError {
+					err: msg,
+					action: Some(msgs::ErrorAction::IgnoreError),
+				},
```

Comment:
is action: None different from Some(IgnoreError)?

Reply:
action: None -> Not yet filled in (ie TODO), action: Some(IgnoreError) -> Just stop procsesing without error.